.vscode/settings.json

```diff
@@ -11,7 +11,7 @@
   "eslint.packageManager": "yarn",
   "eslint.validate": ["vue","html","javascript","typescript"],
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```

Comment:
Yeah we should have been catching this earlier and in a separated issue as it bothered a lot during development, so it would not affect any reverted change.